generator/.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto
+*.sh text eol=lf
+*.md text eol=lf
+*.tmpl text eol=lf
```

Comment:
```suggestion
# Always check-out / check-in files with LF line endings.
* text=auto eol=lf
```

Ok, let's use above to take care of all text files at once (there won't be a need to specify each file type explicitly)